### PR TITLE
Raro 949/doc issues

### PR DESCRIPTION
**Pull request by sgatchell**
The updates the quick start to be more opinionated and sufficient to actually run the ambra stack.
Notes:

- It suggests a quicker path to getting the app running locally.  Alternatives are offered but de-emphasized.
- I emphasized running from source with maven over deploying the .war files to tomcat because I could not get the latter to work.  Probably a personal failure but this ticket is timeboxed so I left in the best state I could.
- Included steps to create the Content Repo DB, which were not present before.
- wombat can be started but it is just an empty page that says: "Congratulations!  You have setup Wombat correctly!"

### CODE REVIEW

File: Quickstart-Guide.md [hunk 1]
```diff
@@ -112,125 +115,84 @@ mysql -uroot -e "DROP DATABASE IF EXISTS repo;"
...
-This will create a directory named `$HOME/ambra/themes`. It will contain:
-
-* a file named `sites.yaml`, which describes the publication sites to be hosted by your server (just one to start); and
-* a directory named `main/`, which is the theme for your one site.
-
-There is no significance to the directory name `main/`, you may change it to anything you wish. You may also change the initial site key in `sites.yaml` to a string that identifies your site.
-
-On a production system, `/var/themes` is recommended instead of `$HOME/ambra/themes`.
-
-### Theme Overrides
```
Comment (sjtower) [inline Quickstart-Guide.md]: Please include a link and a short description of the new `themes.md` here.

File: Quickstart-Guide.md [hunk 2]
```diff
@@ -89,21 +92,21 @@ mysql -uroot -e "DROP DATABASE IF EXISTS ambra;"
...
 ```
 
-Add a journal to the database. For example:
+Add a journal to the database:
 
 ```sql
 INSERT INTO journal (`journalKey`, `title`, `eissn`) VALUES ("my_journal", "My Journal", "0000-0000");
 ```
 
-* The field `journalKey` is an identifier used in config files. It must match the key configured in `journal.yaml` ([see below](#themes-configuration)).
```
Comment (sjtower) [inline Quickstart-Guide.md]: Please specify that `journalKey` is an identifier used in `journal.yaml`, specifically.

File: Quickstart-Guide.md (hunk 1 above)
Comment (sgatchell) [reply Quickstart-Guide.md]: Oops, good catch!

File: Quickstart-Guide.md [hunk 3]
```diff
@@ -89,21 +88,21 @@ mysql -uroot -e "DROP DATABASE IF EXISTS ambra;"
...
-* The field `title` is the reader-facing display form of the title.
-* The field `eissn` is the journal's [electronic ISSN (e-ISSN)](http://www.issn.org/). It must match the e-ISSN for articles ingested into the system (see the [Ingestible Package Guide](https://plos.github.io/ambraproject/Ingestible-Package-Guide.html) for details). On a toy system, use a dummy value such as `0000-0000`.
+* `journalKey` - an identifier used in config files. A theme is set for a journal by setting `journalKey` in `journal.yaml`. For more on themes see the [Themes Quickstart](themes-customizing.html).
```
Comment (sgatchell) [inline Quickstart-Guide.md]: journal.yaml referenced here.

File: Quickstart-Guide.md [hunk 4]
```diff
@@ -112,125 +111,85 @@ mysql -uroot -e "DROP DATABASE IF EXISTS repo;"
...
+cd $HOME
+pwd # This prints the path to your home directory. Edit the file /etc/ambra/wombat.yaml and replace $HOME with this path.  
 ```
 
-## Deploying the artifacts to Tomcat
-
-If you have downloaded or compiled `.war` files, you may deploy them to a Tomcat instance as normal. (Consult the Tomcat documentation for details.)
-
-You may also run each component from Maven. This is where you will set the application port as well as define the configuration directory:
+For more on themes see the [Themes Quickstart](themes-customizing.html) and [Working with PLOS's Themes ](themes-documenation.html).
```
Comment (sgatchell) [inline Quickstart-Guide.md]: links to other themes documentation here:

**Pull request merged**